### PR TITLE
Introduce PriceObserver

### PR DESCRIPTION
**Pull request by gasiortomasz**

### CODE REVIEW

File: universal-login-sdk/lib/core/observers/PriceObserver.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+const cryptocompare = require('cryptocompare');
+
+export type ObservedCurrency = 'USD' | 'EUR' | 'BTC';
+
+export class PriceObserver extends ObserverRunner {
+  private observedTokensSymbols: string[] = [];
+  private lastTokenPrices: {} = {};
+  private callbacks: Function[] = [];
+
+  constructor(private observedTokens: TokenDetails[], private observedCurrencies: ObservedCurrency[], step: number = 1000 * 60) {
```
Comment (marekkirejczyk) [inline universal-login-sdk/lib/core/observers/PriceObserver.ts]: 1000 * 60 -> magic the air!

File: universal-login-sdk/lib/core/observers/PriceObserver.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import {TokenDetails} from '@universal-login/commons';
+import ObserverRunner from './ObserverRunner';
+const cryptocompare = require('cryptocompare');
+
+export type ObservedCurrency = 'USD' | 'EUR' | 'BTC';
+
+export class PriceObserver extends ObserverRunner {
```
Comment (marekkirejczyk) [inline universal-login-sdk/lib/core/observers/PriceObserver.ts]: Isn't PriceObserver the same thing as PriceOracle?

Comment (gasiortomasz) [reply universal-login-sdk/lib/core/observers/PriceObserver.ts]: It is. Thus, I wanted naming to be consistent.

Comment (marekkirejczyk) [reply universal-login-sdk/lib/core/observers/PriceObserver.ts]: Shall we meget those two?

File: universal-login-sdk/lib/core/observers/PriceObserver.ts (hunk 1 above)
Comment (marekkirejczyk) [reply universal-login-sdk/lib/core/observers/PriceObserver.ts]: extract constant `PRICE_OBSERVER_DEAFULT_TICK`

File: universal-login-sdk/test/unit/core/PriceObserver.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    ETH: { USD: 218.21, EUR: 194.38, BTC: 0.01893 },
+    AMB: { USD: 0.02619, EUR: 0.02337, BTC: 0.00000227 }
+  };
+  const pricesAfter = {
+    ETH: { USD: 1838.51, EUR: 1494.71, BTC: 0.09893 },
+    AMB: { USD: 0.2391, EUR: 0.1942, BTC: 0.00001427 }
+  };
+
+  let callCount = 0;
+  const mockedPriceOracle = new PriceObserver(observedTokens, observedCurrencies, 100);
```
Comment (marekkirejczyk) [inline universal-login-sdk/test/unit/core/PriceObserver.ts]: 100 -> TEST_TICK = 5

**Pull request merged**